packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -30,6 +32,7 @@ export const DropdownItem: React.FunctionComponent<DropdownItemProps> = ({
   href = '',
   tooltip = null,
   tooltipProps = {},
+  listItemClassName = '',
```

Comment:
if you default this to null will it still add an empty class tag by default to the `li` elements?

Reply:
Yeah, we could wrap it with `css(listItemClassName)`.

Reply:
Yeah it looks like you're right - thanks for the css() suggestion @redallen